src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java

```diff
@@ -238,4 +238,28 @@
             notificationGenerator
         );
     }
+
+    @Bean
+    public PreSubmitCallbackHandler<AsylumCase> hearingBundleReadyHomeOfficeNotificationHandler(
+            @Qualifier("hearingBundleReadyHomeOfficeNotificationGenerator") NotificationGenerator notificationGenerator) {
+
```

Comment:
Similar to above you can have just one generic handler something like hearingBundleReadyNotificationHandler() as they are  for the same event.